### ISSUE TITLE
onChange Event not working

### ISSUE THREAD

**Issue by wangdawai** [MuhammadUsamaSiddiqui/circular_countdown_timer#55]
I run the example code. No Countdown Changed messge printed.

❯ flutter --version
Flutter 3.10.4 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 682aa387cf (3 days ago) • 2023-06-05 18:04:56 -0500
Engine • revision 2a3401c9bb
Tools • Dart 3.0.3 • DevTools 2.23.1

**Comment by ceranco** [comment]
I ran into the same problem and opened a PR with a fix: https://github.com/MuhammadUsamaSiddiqui/circular_countdown_timer/pull/56
To use it before the PR is merged you can change the entry in your pubspec.yaml to:

```yaml
circular_countdown_timer:
    git:
      url: https://github.com/ceranco/circular_countdown_timer
      ref: on_change_fix
```

**Comment by Abdulah-butt** [comment]
update the package on pub dev too with this fix

**Comment by MuhammadUsamaSiddiqui** [comment]
Hi everyone, I have released a new version **0.2.4** of the package on pub.dev along with this fix and migrating it to the Flutter version 3.22.3
